device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceIO.java

```diff
@@ -121,7 +116,7 @@
 
         this.state = IotHubConnectionStatus.DISCONNECTED;
 
-        this.transport = new IotHubTransport(config, this, isMultiplexing);
+        this.transport = new IotHubTransport(config, this, false);
```

Comment:
It's  always non-multiplexing?

Reply:
For this constructor, yes. That was a warning that this change fixes